src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,117 @@
+              'has-background-info-light': todo.id === currentTodo?.id,
+            })}
+          >
+            <td className="is-vcentered">{todo.id}</td>
             <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">quo adipisci enim quam ut ab</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">9</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">molestiae perspiciatis ipsa</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
+              {todo.completed || (
+                <span className="icon" data-cy="iconCompleted">
+                  <i className="fas fa-check" />
                 </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">10</td>
-
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
+              )}
             </td>
 
             <td className="is-vcentered is-expanded">
-              <p className="has-text-success">
-                illo est ratione doloremque quia maiores aut
+              <p
+                className={classNames(
+                  { 'has-text-danger': todo.completed },
+                  { 'has-text-success': !todo.completed },
+                )}
+              >
+                {todo.title}
               </p>
             </td>
 
             <td className="has-text-right is-vcentered">
               <button data-cy="selectButton" className="button" type="button">
                 <span className="icon">
-                  <i className="far fa-eye" />
+                  <i
+                    className={classNames(
+                      'far',
+                      { 'fa-eye-slash': todo.id === currentTodo?.id },
+                      { 'fa-eye': todo.id !== currentTodo?.id },
+                    )}
+                    onClick={() => setTodo(todo)}
+                  />
                 </span>
               </button>
             </td>
           </tr>
```

Comment:
Consider creating `TableRow` component and use destruction assignment  for `todo` object